arm/Microsoft.Sql/servers/.parameters/parameters.json

```
@@ -18,6 +18,15 @@
                 "secretName": "administratorLoginPassword"
             }
         },
+        "administrators": {
+            "value": {
+                "azureADOnlyAuthentication": false,
+                "login": "John Doe",
+                "sid": "<<objectId>>",
+                "principalType": "User",
```

To work in our environment we ideally set it to the deploying service principal.

```suggestion
                "principalType": "Application",
```